kidkode/src/main/webapp/display.jsp

```diff
@@ -0,0 +1,23 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+    "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Hello World JSP!</title>
+</head>
+<body>
+    <!-- diplaying the text from the ux txt file the server sent-->
+    <c:out value="${text.fileOutput}" />
+    <!-- example of some things we can do with JSP with variables/functions/conditionals!-->
+    <c:set var="str" value="Our team is so cool"/> 
+    <br>
+    <br>
+    <c:if test="${fn:contains(str,'cool')}">
+        this works!
```

Comment:
Right now the example I created only has the description for UX in the text file (picture below), but ideally we'd have the title, description, and links in the text file, parse the text file in the Read File class and make a field for each one of those sections of the page, and then send the object to the JSP file so that we can just easily refer to the fields (like text.link or text.title)  to build the page. Also I included the if tag just as an example of some of the stuff we can do.
<img width="1440" alt="Screen Shot 2020-07-10 at 10 06 49 PM" src="https://user-images.githubusercontent.com/49773006/87214450-a9868780-c2fa-11ea-9e45-ca636b55139a.png">